test/testcafe/spec/IdentifyWithRememberUsername_spec.js

```diff
@@ -225,9 +225,7 @@ test.requestHooks(identifyRequestLogger, identifyMock)('should pre-fill identifi
   await t.expect(identifier).eql('testUsername@okta.com');
 });
 
-
-// OKTA-585939 Cookie is not updated in gen 3 widget
-test.meta('gen3', false).requestHooks(identifyRequestLogger, identifyMock)('should store identifier in ln cookie when updated', async t => {
+test.requestHooks(identifyRequestLogger, identifyMock)('should store identifier in ln cookie when updated', async t => {
```

Comment:
I think I've seen the username field pre-filled in some of our test deployments even before the changes in this PR. Is there something else that was causing the field to be pre-filled?

Reply:
Yeah we also have logic for pre-populating the username field [in the identify transformer](https://github.com/okta/okta-signin-widget/blob/dc0726438fcd15f8d716463f46307259eaf96846/src/v3/src/transformer/identify/transformIdentify.ts#L49C1-L59C1) but I've checked in a preview org and the original bug that Jared fixed for gen2 is present in gen3. I'm not 100% sure how adding this logic to set the cookie upon saving the form fixes the bug, maybe @jaredperreault-okta could give a bit more info?

Reply:
@shawyu-okta chatted with Jared on Slack about the gen2 ticket and got some more info. Was also able to verify that this change should fix the bug from reappearing in gen3